### DISCUSSION TITLE
Tero Subtitler 1.0.1.3

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#201] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.3'>Tero Subtitler 1.0.1.3</a>.</em>

**Comment by torontal7** [comment]
You should have been named the linux zip to be clear that is a 64bit version. I downloaded it and cannot use because I have a 32bit OS.

```
$ ./tero 
tero: error while loading shared libraries: libX11.so.6: cannot open shared object file: No such file or directory
$ file ./tero
tero: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.4.0, stripped
```
Otherwise, why you (and others...) provide only 64bit version of the applications? Why you treat the users of 32bit Linux as non-humans who do not deserve nothing?

**Reply by URUWorks** [reply]
you are right.
we will try to compile for 32bit.

**Comment by ruonghieu** [comment]
Thank you so much for doing this, I have been waiting for a good cross platform subtitling editor, it would be nice if this can incorporate many features from subtitle edit which I am using now, but as it's only for windows therefore I have to use a virtual machine to run it on my m1 mac. I hope you can build a stable working app to run on mac, no need to rush, we just need a good working app that can replace the only decent free app for the mac at the moment which is aegisub which hasn't been updated since 2014 :( thank you so much for this
